modular_snn/modular_snn_config_evaluation.py

```diff
@@ -270,8 +291,10 @@ def ignore_hyperactive_neurons(neuron_spikes, testing_result_monitor, threshold_
                         help='Number of excitatory output neurons. The number of inhibitory neurons are defined the same.')
     parser.add_argument('--threshold_i', type=int, default=0, 
                         help='Threshold value used to ignore the hyperactive neurons.')
+    parser.add_argument('--seed', type=int, default=0, 
```

Comment:
Briefly describe what random numbers are being generated (what for)

Reply:
Updated the help description